front_end/panels/ai_chat/agent_framework/AgentRunner.ts

```
@@ -15,6 +15,7 @@ import { AgentErrorHandler } from '../core/AgentErrorHandler.js';
 import { AgentRunnerEventBus } from './AgentRunnerEventBus.js';
 import { callLLMWithTracing } from '../tools/LLMTracingWrapper.js';
 import { sanitizeMessagesForModel } from '../LLM/MessageSanitizer.js';
+import { FileStorageManager } from '../tools/FileStorageManager.js';
```

ESLint

[error] `../tools/FileStorageManager.js` import should occur before type import of `../tools/Tools.js`

(import/order)